File: 110algoritmos/exercicios006/ex006-Python/ex006.py
# Crie um algoritmo que leia um número e mostre o seu dobro, triplo e raiz quadrada.
num = int(input("Digite um número: "))
dobro = num * 2
triplo = num * 3
raiz = num ** (1/2)
print(f"A o dobro de {num} é {dobro} \nO tripo é {triplo}\nA raiz quadrada é {raiz:.2f}")

File: 110algoritmos/exercicios006/ex006-Elixir/ex006.exs
n = IO.gets("Digite um numero: ")
{num, _} = Float.parse(n)
dobro = num * 2.0
triplo = num * 3.0
raiz = num ** 0.5

IO.puts("O dobro de #{num} eh #{dobro}. \nO triplo eh #{triplo} \nA raiz eh #{Float.round(raiz)}")
